assets/js/modules/analytics/components/setup/SetupForm.test.js

```diff
@@ -302,7 +332,7 @@ describe( 'SetupForm', () => {
 
 		// An additional wait is required in order for all resolvers to finish.
 		await act( async () => {
-			await waitForDefaultTimeouts();
+			await waitForTimeouts( 50 );
```

Comment:
Very minor, but this looks like an arbitrary value so we might as well use a timeout of `30` here to be consistent with the other places we're using `waitForTimeouts()`, unless `50` is explicitly needed here.

```suggestion
			await waitForTimeouts( 30 );
```